campaigns.json

```diff
@@ -1336,8 +1384,7 @@
         ]
       }, 
       "/MinBias_TuneCP5_13TeV-pythia8/RunIISummer19UL16SIM-106X_mcRun2_asymptotic_v3-v1/GEN-SIM": {
-        "SiteWhitelist": [
-          "T1_UK_RAL_Disk", 
```

Comment:
Why do you remove RAL?

Reply:
Because this is only on tape at FNAL and I thought if it needed to be transferred to disk FNAL would be easier.